openassessment/workflow/models.py

```diff
@@ -311,6 +312,9 @@ def update_from_assessments(self, assessment_requirements, override_submitter_re
         if self.status == self.STATUS.cancelled:
             return
 
+        if self.status == self.STATUS.returned:
```

Comment:
Seems we can modify the previous condition to checking, that status in some list.

Reply:
Done